# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBObjectStore.cpp
# Commit: 9966f5d71bb2
# Full Hash: 9966f5d71bb2db2228a914743dca457dcc337d02
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Apply renamings suggested for Cursor::*Key members and related identifiers. r=ytausky,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43249
# ==============================================================================

diff -r 714d36c43204 -r 9966f5d71bb2 dom/indexedDB/IDBObjectStore.cpp
--- a/dom/indexedDB/IDBObjectStore.cpp	Wed Sep 18 06:41:23 2019 +0000
+++ b/dom/indexedDB/IDBObjectStore.cpp	Wed Sep 18 06:41:35 2019 +0000
@@ -73,7 +73,7 @@
   indexUpdateInfo.value() = aKey;
   if (!aLocale.IsEmpty()) {
     const auto result =
-        aKey.ToLocaleBasedKey(indexUpdateInfo.localizedValue(), aLocale, *aRv);
+        aKey.ToLocaleAwareKey(indexUpdateInfo.localizedValue(), aLocale, *aRv);
     if (result.Is(Invalid, *aRv)) {
       aRv->Throw(NS_ERROR_DOM_INDEXEDDB_UNKNOWN_ERR);
     }